boa_parser/src/lexer/mod.rs

```diff
@@ -89,31 +89,29 @@ impl<R> Lexer<R> {
     }
 
     /// Sets the goal symbol for the lexer.
-    #[inline]
+
     pub(crate) fn set_goal(&mut self, elm: InputElement) {
         self.goal_symbol = elm;
     }
 
     /// Gets the goal symbol the lexer is currently using.
-    #[inline]
+
     pub(crate) const fn get_goal(&self) -> InputElement {
         self.goal_symbol
     }
 
-    #[inline]
     /// Returns if strict mode is currently active.
     pub(super) const fn strict_mode(&self) -> bool {
         self.cursor.strict_mode()
     }
 
-    #[inline]
     /// Sets the current strict mode.
     pub(super) fn set_strict_mode(&mut self, strict_mode: bool) {
         self.cursor.set_strict_mode(strict_mode);
     }
 
     /// Creates a new lexer.
-    #[inline]
+
```

Comment:
Don't like the empty line between the comment and function
2 more in lines 92 and 98